app/src/main/java/com/chartiq/chartiqsample/MainActivity.java

```diff
@@ -64,9 +64,8 @@
     private static final int DRAW_REQUEST_CODE = 1;
     private static final int STUDIES_REQUEST_CODE = 2;
     private static final int CHART_OPTIONS_REQUEST_CODE = 3;
-    private static final int REFRESH_INTERVAL = 1;
     private static final String defaultSymbol = "AAPL";
-    public static final String chartUrl ="http://yourdeployment/sample-template-native-sdk.html";
+	public static final String chartUrl ="http://yourdeployment/sample-template-native-sdk.html";
```

Comment:
I wouldn't have picked on this but since I added the other review comment, change this to spaces, instead of a tab.